rio/src/screen/mouse.rs

```diff
@@ -3,6 +3,7 @@ use crate::crosswords::pos::Line;
 use crate::crosswords::pos::Side;
 use crate::event::ClickState;
 use crate::screen::Pos;
+use std::rc::Rc;
```

Comment:
I think this import isn't necessary anymore.

Reply:
deleted. 38271fcee8cfa52a897150708224820cc9f19a09